great_expectations/rule_based_profiler/parameter_builder/value_set_multi_batch_parameter_builder.py

```diff
@@ -95,35 +95,16 @@ def _build_parameters(
                 domain=domain,
                 parameter_reference=self.fully_qualified_parameter_name,
                 expected_return_type=None,
-                parameters={
-                    domain.id: parameter_container,
```

Comment:
Glad this is gone, but it still seems useful in tests.

Reply:
@NathanFarmer I modified the tests so as to drive this object as if in the actual `Rule` scenario -- makes things consistent and simple this way -- that was the point of the PR.  Thanks.